modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptFetchClientCodegen.java

```diff
@@ -316,6 +331,16 @@ public void processOpts() {
         setGenerateValidationAttributes(convertPropertyToBooleanAndWriteBack(VALIDATION_ATTRIBUTES));
     }
 
+    @Override
+    public String apiDocFileFolder() {
+        return (outputFolder + File.separator + apiDocPath);
```

Comment:
...because we seem to be using those constantly here for a local path. If we want to use the constants both in a file path context and a web context we should remove the trailing slash and add it when we use them in a web context.